### ISSUE TITLE
bug: Preprocessing from mdbook 0.5.2 fails with wrong JSON emitted

### ISSUE THREAD

**Issue by lukem** [joshrotenberg/mdbook-lint#365]
Using **mdbook-lint** 0.13.6 as a preprocessor to **mdbook** 0.5.2 fails with:
```
 INFO Book building has started
mdbook-lint: No issues found
ERROR Unable to parse the preprocessed book from "lint" processor
	Caused by: missing field `items` at line 1 column 94
```

As far as I can tell, **mdbook** 0.5.2 expects the Book JSON object to have "items" member but **mdbook-lint** renames "items" to "sections" (the older name from **mdbook** 0.4.x?)  in `crates/mdbook-lint-cli/src/preprocessor.rs` function `normalize_mdbook_json()`

The **mdbook** change was made before **mdbook** 0.5.0 (see **mdbook** pull request https://github.com/rust-lang/mdBook/pull/2813/commits/800fb54aeb2ed5132d584de635c377b951cc8133)
